spikesorters/basesorter.py

```diff
@@ -86,7 +86,15 @@ def __init__(self, recording=None, output_folder=None, verbose=False,
             self.recording_list = recording.get_sub_extractors_by_property(grouping_property)
             n_group = len(self.recording_list)
             self.output_folders = [output_folder / str(i) for i in range(n_group)]
-
+            
+        # make dummy location if no location because some sorter need it
+        for recording in self.recording_list:
+            print('WARNING! No channel location given. Make dummy location.')
```

Comment:
This print should go after the next if

Make dummy location --> Adding dummy locations